### ISSUE TITLE
Serverside Sync?

### ISSUE THREAD

**Issue by CruelAlpha** [WSIContractor/fivem_doors#1]
Hey!
I was wondering if it were possible to bring in server sync to this?
Would be great!

**Comment by WSIContractor** [comment]
Yes, I am going to work on that shortly. As well as a few other mods I have in mind.

Nice that you noticed!

Cheers!

**Comment by CruelAlpha** [comment]
Haha np!
On my end i edited the script so its a special key combination rather than everyone knowing its "E"
I setup multiple key combinations so its a bit more secure and only cops would know
Thanks!

**Comment by Darklandz** [comment]
Thanks for taking my code, you just changed my name to yours and claim you made it. i updated MY code on the fivem forums to make it work server side. you gonna claim you made that to?

I have no problems when others use my code, at least state who the original author was if you use and change it (wich you did not, you just put your name on it)

At least have some respect and don't claim something is yours when it's NOT.
